### ISSUE TITLE
new github token UI

### ISSUE THREAD

**Issue by evanrichter** [mayhemheroes/hackathon-resources#11]
the screenshot for lab 2b, step 6 is slightly out of date with GitHub's new fine-grained token beta

The steps should now be

1. click Personal access tokens
2. click Tokens (classic)
3. then top-right will have the generate token dialog

---

A separate issue would be to try the new fine-grained tokens feature, but maybe not until it's out of beta.

**Comment by xansec** [comment]
Thanks, fixed -> https://github.com/mayhemheroes/hackathon-resources/commit/013fff7765207c948bf7476e15fd1ec1fbac7426
